layouts/millennium/r4/sidebar.html

```diff
@@ -90,6 +90,7 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
         <ul class="js-guides">
           <li class="subtopic"><a href="/millennium/r4/financial/support/">Support</a></li>
           <li><a href="/millennium/r4/financial/support/coverage/">Coverage</a></li>
+          <li><a href="/millennium/r4/financial/support/insurance-plan/">InsurancePlan</a></li>
```

Comment:
Need to change the path as per Bhavya's comment.

Reply:
done https://github.com/cerner/fhir.cerner.com/pull/809/commits/f285387f72f531a780abd964a8c1b3335e9d5385#diff-db70aef4601cdc7697917f05be2110c0c9e05c4346f1e0c64a137e344b3d3bd6R98